INSTALL.md

```diff
@@ -213,62 +237,76 @@ filling in this section of the README.
 
 ### Linux / Windows
 
-Once all the prerequisites are installed, compiling Alacritty should be easy:
-
 ```sh
 cargo build --release
 ```
 
 If all goes well, this should place a binary at `target/release/alacritty`.
 On Windows this directory should also contain the `winpty-agent.exe`.
 
-#### Desktop Entry
-
-Many linux distributions support desktop entries for adding applications to
-system menus. To install the desktop entry for Alacritty, run
+### MacOS
 
 ```sh
-sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
-sudo cp extra/logo/alacritty-term.svg /usr/share/pixmaps/Alacritty.svg
-sudo desktop-file-install extra/linux/Alacritty.desktop
-sudo update-desktop-database
+make app
+cp -r target/release/osx/Alacritty.app /Applications/
 ```
 
-You can find a prerendered SVG logo as well as simplified versions of the SVG in
-the `extra/logo/compat` directory.
+## Post Build
 
-### MacOS
+There are some extra things you might want to set up after installing Alacritty.
+All the post build instruction assume you're still inside the Alacritty
+repository.
```

Comment:
This is a tiny point, but I would think, ... inside the `alacritty/` directory is a bit more clear.

Reply:
No, that would just be more confusing because there's an `alacritty` directory in the repo and you can name the clone whatever you want. So saying you should be inside the repo makes most sense.